src/LinearLoader/index.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+import propTypes from 'prop-types'
+import cx from 'classnames'
+import { theme } from '../theme.js'
+
+const Progress = ({ amount }) => {
+    return (
+        <div>
+            <style jsx>{`
+                div {
+                    width: ${amount}%;
+                }
+            `}</style>
+            <style jsx>{`
```

Comment:
The two style tags can be combined to one

Reply:
I think the current implementation is following the styled-jsx guidelines:
> That said when your CSS is mostly static we recommend to split it up in static and dynamic styles and use two separate style tags so that, when changing, only the dynamic parts are recomputed/rendered.

https://github.com/zeit/styled-jsx#dynamic-styles

Reply:
Yes, dynamic styles should be in a separate `style` element. It's dynamic because it uses a `prop` and cannot be resolved through a constant.